packages/api/lib/granules.js

```diff
@@ -125,7 +125,11 @@ async function moveGranuleFilesAndUpdateDatastore(params) {
       });
     } catch (error) {
       updatedFiles.push({ ...file, fileName: getNameOfFile(file) });
-      log.error(`Failed to move file ${JSON.stringify(moveFileParam)} -- ${JSON.stringify(error.message)}`);
```

Comment:
Nit: I generally don't mind opinionated linters, but this change feels..... icky?     This doesn't feel like an improvment to readability *at all*.

Reply:
https://github.com/nasa/cumulus/pull/3225/commits/ea12a2846db17b78e9107a5109a445018d7ddfc8 

The problem here is that I had the wrong line length set.